xml/System.Text.Json/JsonValueKind.xml

```diff
@@ -54,7 +54,7 @@
       </ReturnValue>
       <MemberValue>6</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that a value is the JSON value <see langword="false" />.</summary>
```

Comment:
@rpetrusha - looks like you missed updating "False" in your effort to "shorten the text".